servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/LoadBalancerBuilder.java

```diff
@@ -104,6 +104,19 @@ LoadBalancerBuilder<ResolvedAddress, C> loadBalancerObserver(
     LoadBalancerBuilder<ResolvedAddress, C> connectionSelectorPolicy(
             ConnectionSelectorPolicy<C> connectionSelectorPolicy);
 
+    /**
+     * Set the maximum number of healthy backends to load balance against using a random-subsetting strategy.
+     *
+     * Note: If a backend within the subset is found to be unhealthy, another endpoint will be added until the unhealthy
+     * backend recovers, after which the subset will return to its original state.
+     *
+     * @param maxUsed the maximum number of healthy backends to use.
+     * @return {@code this}
+     */
+    default LoadBalancerBuilder<ResolvedAddress, C> maxRandomSubsetSize(int maxUsed) {
```

Comment:
One life hack that I learned is to first add a method without default impl, then compiler finds all implementations where I need to implement it (like `DelegatingLoadBalancerBuilder`), then I add the default impl.